src/test/java/uk/gov/ons/census/casesvc/service/UacProcessorTest.java

```diff
@@ -16,23 +20,31 @@
 import org.springframework.amqp.rabbit.core.RabbitTemplate;
 import org.springframework.test.util.ReflectionTestUtils;
 import uk.gov.ons.census.casesvc.client.UacQidServiceClient;
+import uk.gov.ons.census.casesvc.logging.EventLogger;
 import uk.gov.ons.census.casesvc.model.dto.ResponseManagementEvent;
 import uk.gov.ons.census.casesvc.model.dto.UacQidDTO;
 import uk.gov.ons.census.casesvc.model.entity.Case;
 import uk.gov.ons.census.casesvc.model.entity.UacQidLink;
+import uk.gov.ons.census.casesvc.model.repository.CaseRepository;
 import uk.gov.ons.census.casesvc.model.repository.UacQidLinkRepository;
 
 @RunWith(MockitoJUnitRunner.class)
 public class UacProcessorTest {
 
   @Mock UacQidLinkRepository uacQidLinkRepository;
 
+  @Mock CaseRepository caseRepository;
+
   @Mock RabbitTemplate rabbitTemplate;
 
   @Mock UacQidServiceClient uacQidServiceClient;
 
+  @Mock EventLogger eventLogger;
+
   @InjectMocks UacProcessor underTest;
 
+  public static final EasyRandom easyRandom = new EasyRandom();
```

Comment:
petty, but does this need to public

Reply:
It's actually not even used anymore so I'll get rid.